mkdocs.yml

```diff
@@ -134,10 +134,11 @@ extra:
     mpsAPI: 2.1.0
     rpsAPI: 2.1.0
     oamtct: 2.1.0
-    rpc_go: 2.1.0
+    rpc_go: 2.0.0
     rpc_c: 2.0.0
```

Comment:
do we want one for mps router?

Reply:
Not needed for now I'm thinking, this is just for being able to update any links/clone cmds in one place.  MPS Router individually isn't referenced anywhere.  Just as part of the full open-amt-cloud-toolkit repo.

Don't know if I'd call that a 'gap' necessarily.  Is there any docs that could be needed specifically for MPS Router?

Reply:
Nothing from a configuration standpoint.  In the future might be good to add one just as an overview.  Would be nice to have at least an overview for all components.